convert/github/convert.go

```diff
@@ -12,22 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package github converts Github pipelines to Harness pipelines.
+// Package github converts GitHub pipelines to Harness pipelines.
 package github
 
 import (
 	"bytes"
-	"errors"
+	"fmt"
 	"io"
 	"os"
+	"regexp"
+	"strconv"
+	"strings"
 
+	v1 "github.com/drone/go-convert/convert/github/yaml"
```

Comment:
I would recommend changing `v1` to `github`

```diff
v1 "github.com/drone/go-convert/convert/github/yaml"
github "github.com/drone/go-convert/convert/github/yaml"
```

we use `v0` and `v1` when both yaml parsers are for the same vendor (e.g. harness v0 to harness v1 or harness/drone to harness v1)

Reply:
Totally, I original had GitHub but for whatever reason I changed it back 😓